.gitignore

```diff
@@ -19,3 +19,4 @@ sam.yaml
 
 .env
 .eslintcache
+.schemaTree.json
```

Comment:
Where does this file come from?

Reply:
This was from local testing, I will delete that